drivers/serial/uart_mcux_flexcomm.c

```diff
@@ -1124,6 +1124,7 @@ static int mcux_flexcomm_pm_action(const struct device *dev, enum pm_device_acti
 
 	usart_intenset = USART_GetEnabledInterrupts(config->base);
 
+	int ret;
```

Comment:
I think this is going to fail check compliance script on CI, or I would be surprised if it doesn't, please move to line 1126 and put the newline after the declaration instead of before

Reply:
Can do. Which specific check / guideline would you expect this to violate?

Reply:
I made the change again after your edit, but I'm still curious to know which compliance check is relevant here. I don't see anything about declarations being at the beginning of the function in https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html and AFAICT Zephyr doesn't strictly follow C89. But also, I'm new to the codebase so 🤷🏽‍♂️

Reply:
Ah I see thanks